torchani/data/new.py

```diff
@@ -0,0 +1,448 @@
+import numpy as np
+import torch
+import functools
+from ._pyanitools import anidataloader
+import importlib
+PKBAR_INSTALLED = importlib.util.find_spec('pkbar') is not None
+if PKBAR_INSTALLED is not None:
```

Comment:
This is wrong

Reply:
what do you mean

Reply:
`PKBAR_INSTALLED` is a boolean, why are you checking if it is `None`?

Reply:
Did you test it before you comment?
```
print(importlib.util.find_spec('pkbar'))
ModuleSpec(name='pkbar', loader=<_frozen_importlib_external.SourceFileLoader object at 0x7fe0c5f457b8>, origin='/home/richard/program/anaconda3/envs/ml/lib/python3.6/site-packages/pkbar/__init__.py', submodule_search_locations=['/home/richard/program/anaconda3/envs/ml/lib/python3.6/site-packages/pkbar'])
```
```
print(importlib.util.find_spec('hello'))
None
```

Reply:
Did you tested it **CORRECTLY**? You should 
```python
print(importlib.util.find_spec('pkbar') is not None)
```